_inc/client/at-a-glance/index.jsx

```diff
@@ -69,7 +69,10 @@ class AtAGlance extends Component {
 		);
 		const connections = (
 			<div>
-				<DashSectionHeader label={ __( 'Connections' ) } />
+				<DashSectionHeader
+					label={ __( 'Connections' ) }
+					className="jp-dash-section-header__connections"
```

Comment:
Should this use the `jp-at-a-glance` prefix instead of `jp-dash`, like the rest of the classes on that page?

Reply:
I went with the naming on the `DashSectionHeader` component instead of the entire page: https://github.com/Automattic/jetpack/blob/c85b94d13b3926cb217d8282aab9c1e8a714164e/_inc/client/components/dash-section-header/index.jsx#L39

Reference: https://wpcalypso.wordpress.com/devdocs/docs/coding-guidelines/css.md